src/core/Vector.re

```diff
@@ -1,5 +1,6 @@
 // The core vector type.
 type vector('a) = ('a, 'a);
+type t('a) = vector('a);
```

Comment:
`type t` is a special concept in Reason / OCaml. Typically it is used to refer to the type of a module itself. In this case, the `type t` of the `Vector` module is the `vector` type itself.